### PR TITLE
fix: android getting started for react native push notifications

### PR DESCRIPTION
**Pull request by tannerabread**
_Issue #, if available:_

_Description of changes:_
This fix removes step 8 which explicitly pinned the gradle distribution at 6.3 which causes issues and is an unnecessary step. It also adds a callout about android 12+ versions.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by abdallahshaban557 — DISMISSED** [review]
LGTM

File: src/fragments/lib/push-notifications/js/getting-started.mdx [hunk 1]
```diff
@@ -91,13 +91,13 @@ Setup instructions are provided for Android and iOS, and configuration for both
...
-8. Open *android/gradle/wrapper/gradle-wrapper.properties* update the Gradle `distributionUrl`:
+8. Open *android/app/src/main/AndroidManifest.xml* file and add the following configuration into `application` element.
 
-    ```properties
-    distributionUrl=https\://services.gradle.org/distributions/gradle-6.3-all.zip
-    ```
+    <Callout>
 
-9. Open *android/app/src/main/AndroidManifest.xml* file and add the following configuration into `application` element.
+    When using Android 12+, make sure to add the property `android:exported="false"` to the service tags below.
```
Comment (cshfang) [inline src/fragments/lib/push-notifications/js/getting-started.mdx:98]: I think we might be able to just add that property to our snippet. Android 12 began enforcing that this property be explicitly specified but there should be no harm (and is perhaps better practice) to include it regardless.

Comment (tannerabread) [reply src/fragments/lib/push-notifications/js/getting-started.mdx]: Changed this to suggested

**Review by cshfang — APPROVED** [review]
Lgtm

**Review by abdallahshaban557 — APPROVED** [review]
LGTM

**Pull request merged**